### PR TITLE
concurrent builds test cakephp only

### PR DESCRIPTION
**Pull request by lenahorsley**

### CODE REVIEW

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-ci-tests-conc-builds-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-ci-tests-conc-builds-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-ci-tests-conc-builds-kube-burner

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -399,7 +399,86 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
+<<<<<<< HEAD
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:402]: looks like you had a small issue with your merge. think 402-404 can be deleted

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 2]
```diff
@@ -399,7 +399,86 @@ periodics:
...
+        secretName: cluster-secrets-aws-perfscale-qe
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+>>>>>>> 8e783c2d580 (concurrent builds test cakephp only)
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:481]: this line as wlel should be deleted

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-ci-tests-conc-builds-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-ci-tests-conc-builds-kube-burner

**Pull request merged**